README.md

```diff
@@ -68,6 +68,22 @@ module "cxm-integration" {
 }
 ```
 
+### EKS Cluster Enablement
+
+For enabling CXM access to existing EKS clusters, use the dedicated EKS cluster enablement module:
+
+```hcl
+module "cxm_eks_enablement" {
+  source = "./terraform-aws-eks-cluster-enablement"
+
+  cluster_name = "my-production-cluster"
+  # Use the IAM role from the CXM integration module
+  iam_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${module.cxm-integration.iam_role_name}"  # Construct ARN from account ID and role name
```

Comment:
We need to simplify that. By outputing the arn in the integration module or allowing to pass the role_name only as the eks module knows which account it is set under.